daemon/graphdriver/devmapper/README.md

```diff
@@ -60,8 +62,10 @@ status information about the driver.
  *  `Metadata file` blockdevice file used for the devicemapper metadata
  *  `Data Space Used` tells how much of `Data file` is currently used
  *  `Data Space Total` tells max size the `Data file`
+ *  `Data Space Available` tells how much free space there is in the `Data file`
```

Comment:
Perhaps, lets add a parenthetical wording regarding the adjustment made for underlying filesystem space, when the device is a loop file